WordPress/Classes/ViewRelated/Blog/Site Picker/BlogList/BlogListReducer.swift

```diff
@@ -0,0 +1,183 @@
+enum BlogListReducer {
```

Comment:
That is indeed odd. There's a practice as defining stateless types as enum. The reason is that you cannot initiate an enum by mistake. Whereas if you define it as struct or class, one might try to use it by initiating one and accessing member functions.

I was never able to fully internalize this as it feels like misusing `enum` but from what I know, in a technical perspective enum is a better fit due to the reason I mentioned. All it does here is to provide a namespace. Struct offers more than that which I don't really want here.

The same applies to `enum Constants` that you might see all around the codebase.

Reply:
That's interesting. It looks like there should be some "static class" for this purpose. Probably, `class BlogListReducer` with static functions and properties would have the same effect, but, as you mentioned, it can be initiated by mistake. Anyway, "today I learnt". 😄